### DISCUSSION TITLE
What is the difference?

### DISCUSSION THREAD

**Discussion by eyyyyyyy3** [tauri-apps/tauri#7755] [Q&A]
I recently saw a poste where someone used ** in their scope definiton. So instead of 
` "fs": {
        "all": true,
        "scope": [
          "$RESOURCE/*",
........
`

he wrote 
` "fs": {
        "all": true,
        "scope": [
          "$RESOURCE/**",
........
`

what is the difference between these two?

**Comment by M69W** [comment]
`/*` and `/**`

/* is to intercept all folders, excluding subfolders
/** is to block all folders and subfolders inside

Equivalent to /* only one level behind

/** Can contain multiple levels.

**Reply by eyyyyyyy3** [reply]
What do you mean by „block all folders“? I thought you include them with `“scope”: `
